lapis/src/main/kotlin/org/genspectrum/lapis/response/LapisResponse.kt

```diff
@@ -24,29 +24,29 @@ private const val REPORT_TO =
 
 @Schema(description = LAPIS_INFO_DESCRIPTION)
 data class LapisInfo(
-    @Schema(
+    @field:Schema(
```

Comment:
so these will now only be applied to the field?

Reply:
Yes.